<launch>
  <arg name="smach_topics" default="/server_name/smach/container_structure /server_name/smach/container_status" />
  <arg name="marker_topics" default="/visualize_objectdetection_marker_array /visualization_marker /object_detection_marker_array /move_base_marker_array /spots_marker_array /spots_pictogram /kinect_head_c2/rgb/masked_objects_marker" />
  <arg name="mask_topics" default="/kinect_head_c2/rgb/mask_fridge /kinect_head_c2/rgb/mask_drawer /kinect_head_c2/rgb/mask_table /kinect_head_c2/rgb/mask_door_left /kinect_head_c2/rgb/mask_door_right" />
  <arg name="navigation_topics" default="/move_base_node/local_costmap/costmap /move_base_node/global_costmap/costmap /eng2/7f" />

  <include file="$(find jsk_data)/launch/pr2_record.launch">
    <arg name="other_topics" value="$(arg mask_topics) $(arg navigation_topics) $(arg smach_topics) $(arg marker_topics) /tf" />
    <arg name="other_options" value=""/>

    <!-- for PR2 -->
    <arg name="save_openni" value="true" />
    <arg name="save_tilt_scan" value="true" />
    <arg name="save_base_scan" value="true" />
    <arg name="save_robot_model" value="true" />
    <arg name="save_all_image" value="false" />
    <arg name="save_pressure" value="true" />
    <arg name="save_gripper_command" value="true" />
    <arg name="camera_namespace" value="kinect_head"/>
    <arg name="save_dir" value="/removable/jsk_pr2_wandering/data"/>
    <arg name="remove_c2" value="false" />
  </include>

</launch>
